Register polling tentacle (#990)
OctopusDeploy/docs: docs/octopus-rest-api/examples/register-polling-tentacle.md

---

> title: Register polling tentacle

Reviewer: ```suggestion
title: Register Polling Tentacle
```

> description: An example script to register a polling tentacle.

Reviewer: ```suggestion
description: An example script to register a Polling Tentacle.
```

---

> ## Description

Reviewer: ```suggestion
```

> These scrips demonstrate how to programmatically add a polling tentacle.  Polling tentacles usually self-register during the installation process, but in cases where the tentacle was deleted from the server and need to be readded, these scripts will show you how.

Reviewer: ```suggestion
These scrips demonstrate how to programmatically add a Polling Tentacle.  Polling Tentacles usually self-register during the installation process, but in cases where the Tentacle was deleted from the server and needs to be re-added, these scripts will show you how.
```
